core/data/src/main/java/team/ppac/data/repository/SampleRepositoryImpl.kt

```diff
@@ -0,0 +1,16 @@
+package team.ppac.data.repository
+
+import team.ppac.domain.model.SampleImageModel
+import team.ppac.domain.repository.SampleRepository
+import team.ppac.remote.datasource.SampleDataSource
+import javax.inject.Inject
+
+class SampleRepositoryImpl @Inject constructor(
+    private val sampleDataSource: SampleDataSource
+) : SampleRepository {
+    override suspend fun getImages(): List<SampleImageModel> {
+        return sampleDataSource.getImages().map { sampleResponse ->
+            SampleImageModel(sampleResponse.url)
+        }
```

Comment:
```suggestion
        return sampleDataSource.getImages().toDomain()
        
        internal fun List<SampleResponse>.toDomain(): List<SampleImageModel> {
            return this.map { sampleResponse ->
                SampleImageModel(sampleResponse.url)
            }
        }
```
지금은 모델에 들어가는게 하나밖에 없어서 상관없을 것 같긴한데, 여러개 들어가게 되면 이렇게 따로 Mapper 확장함수 만들어서 쓰는게 가독성에 더 좋아보여용~ @.@

Reply:
일단은 샘플이라 저렇게 했습니다..! 기능 구현 때는 Mapper 활용해볼게요~